.github/workflows/review.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           fi
       - name: Run eslint
         if: env.run_eslint == 1
-        uses: reviewdog/action-eslint@v1
+        uses: reviewdog/action-eslint@v1.12.1
```

Comment:
@ilyam8 already made comments related to this update, please, take a look here too, whether it is necessary to change.

Reply:
Actually, this doesn't work, I was being optimistic. I'm going to drop this all around.

I'll have to spend time w/ another hat figuring out why the action is flaking (act/reviewdog). I spent a bit of time looking into it last night.